src/app/modules/admin/components/profile-header/profile-header.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit } from '@angular/core';
+import { DomSanitizer } from '@angular/platform-browser';
+import { select, Store } from '@ngrx/store';
+import { Observable } from 'rxjs';
+import { AppState } from 'src/app/store/app.state';
+import { selectAvatar, selectName } from '../../state/admin.selectors';
+
+@Component({
+    selector: 'app-profile-header',
+    templateUrl: './profile-header.component.html',
+    styleUrls: ['./profile-header.component.scss'],
+})
+export class ProfileHeaderComponent implements OnInit {
+    constructor(private store: Store<AppState>) {}
+
+    name$: Observable<any>;
+    avatar$: Observable<any>;
+    profileAvatar: string;
+
+    ngOnInit(): void {
+        this.name$ = this.store.pipe(select(selectName));
+        this.avatar$ = this.store.pipe(select(selectAvatar));
```

Comment:
move to the top

Reply:
you should import all admin selectors as AdminSelectors and then use AdminSelectors.selectName - it's easier to read